addon-test-support/scroll.js

```diff
@@ -0,0 +1,47 @@
+import getElementWithAssert from './-private/get-element-with-assert';
+import wait from 'ember-test-helpers/wait';
+
+
+/*
+  Triggers a paint (and therefore flushes any queued up scroll events).
+
+  @method triggerFlushWithPromise
+  @return {RSVP.Promise}
+  @private
+*/
+function triggerFlushWithPromise(){
+  return wait().then(() => {
+    return new Ember.RSVP.Promise((resolve) => {
+      window.requestAnimationFrame(resolve);
+    });
+  });
+}
+
+/*
+  Scrolls DOM element or selector to the given position (if the DOM element is currently overflowed).
+  The promise resolves after the scroll event has been triggered.
+  @method scrollLeft
+  @param {String|HTMLElement} selector
+  @param {Number} position
+  @return {RSVP.Promise}
+  @public
+*/
+export function scrollLeft(selector, position) {
+  getElementWithAssert(selector).scrollLeft = position;
+  return triggerFlushWithPromise();
+}
+
+/*
+  Scrolls DOM element or selector to the given position (if the DOM element is currently overflowed).
+  The promise resolves after the scroll event has been triggered.
+
+  @method scrollTop
+  @param {String|HTMLElement} selector
+  @param {Number} position
+  @return {RSVP.Promise}
+  @public
+*/
```

Comment:
I think we should be copying the setter methods `Element.scrollTop` and `Element.scrollLeft` instead of `Window.scrollTo`